conf/defaults.ini

```diff
@@ -406,6 +406,9 @@ csrf_always_check = false
 # Comma-separated list of plugins ids that won't be loaded inside the frontend sandbox
 disable_frontend_sandbox_for_plugins = grafana-incident-app
 
+# Regex match for POST URL in actions. Empty will allow anything that is not on the same origin
```

Comment:
I think @dannyc-grafana mentioned that we should rather use paths instead of regex.

Reply:
yeah this comment is out of date.

we discussed maybe having allowed url list per verb (method), and maybe prefix or glob matching at some later time.